kata-containers.target

```diff
@@ -7,6 +7,7 @@
 [Unit]
 Description=Kata Containers Agent Target
 Requires=basic.target
+Requires=tmp.mount
```

Comment:
I thought we deleted this unit?

https://github.com/kata-containers/osbuilder/pull/290/files#diff-7b8891154976b961e2cd7166dc005c90R59

Reply:
Ah - and now I find that we did, but we're re-adding it on https://github.com/kata-containers/osbuilder/pull/301 :)